get_content_types() not working. throwing AttributeError

I'm receiving this for nearly all endpoint in schemas.py in get_content_types()

```python
>   content_types = [list(definition.keys()) for definition in definitions]
E   AttributeError: 'str' object has no attribute 'keys'
```

probably has something to do that in `definitions = responses.get(str(response.status_code), {}).values()` .values() returns a dict_values object and not dict object

schemathesis 1.0.3, python 3.8.1

--- a/docs/changelog.rst
+++ b/docs/changelog.rst
@@ -6,6 +6,11 @@ Changelog
 `Unreleased`_
 -------------
 
+Fixed
+~~~~~
+
+- ``OpenApi30.get_content_types`` behavior, introduced in `8aeee1a <https://github.com/kiwicom/schemathesis/commit/8aeee1ab2c6c97d94272dde4790f5efac3951aed>`_. `#469`_
+
 `1.0.3`_ - 2020-04-03
 ---------------------
 
@@ -841,6 +846,7 @@ Fixed
 .. _0.3.0: https://github.com/kiwicom/schemathesis/compare/v0.2.0...v0.3.0
 .. _0.2.0: https://github.com/kiwicom/schemathesis/compare/v0.1.0...v0.2.0
 
+.. _#469: https://github.com/kiwicom/schemathesis/issues/469
 .. _#463: https://github.com/kiwicom/schemathesis/issues/463
 .. _#461: https://github.com/kiwicom/schemathesis/issues/461
 .. _#457: https://github.com/kiwicom/schemathesis/issues/457

--- a/src/schemathesis/schemas.py
+++ b/src/schemathesis/schemas.py
@@ -415,9 +415,8 @@ class OpenApi30(SwaggerV20):  # pylint: disable=too-many-ancestors
         except KeyError:
             # Possible to get if `validate_schema=False` is passed during schema creation
             raise InvalidSchema("Schema parsing failed. Please check your schema.")
-        definitions = responses.get(str(response.status_code), {}).values()
-        content_types = [list(definition.keys()) for definition in definitions]
-        return list(itertools.chain(*content_types))
+        definitions = responses.get(str(response.status_code), {}).get("content", {})
+        return list(definitions.keys())
 
 
 def get_common_parameters(methods: Dict[str, Any]) -> List[Dict[str, Any]]:

--- a/test/runner/test_checks.py
+++ b/test/runner/test_checks.py
@@ -77,15 +77,14 @@ def test_content_type_conformance_valid(spec, response, case):
     assert content_type_conformance(response, case) is None
 
 
-@pytest.mark.parametrize("content_type, is_error", (("application/json", False), ("application/xml", True),))
-def test_global_produces_override(content_type, is_error):
-    # When "produces" is specified on the schema level and on the operation level
-    schema = schemathesis.from_dict(
+@pytest.mark.parametrize(
+    "raw_schema",
+    (
         {
             "swagger": "2.0",
             "info": {"title": "Sample API", "description": "API description in Markdown.", "version": "1.0.0"},
             "host": "api.example.com",
-            "basePath": "/v1",
+            "basePath": "/",
             "schemes": ["https"],
             "produces": ["application/xml"],
             "paths": {
@@ -98,9 +97,29 @@ def test_global_produces_override(content_type, is_error):
                     }
                 }
             },
-        }
-    )
-    endpoint = schema.endpoints["/v1/users"]["get"]
+        },
+        {
+            "openapi": "3.0.2",
+            "info": {"title": "Test", "description": "Test", "version": "0.1.0"},
+            "paths": {
+                "/users": {
+                    "get": {
+                        "responses": {
+                            "200": {
+                                "description": "OK",
+                                "content": {"application/json": {"schema": {"type": "object"}}},
+                            }
+                        },
+                    }
+                }
+            },
+        },
+    ),
+)
+@pytest.mark.parametrize("content_type, is_error", (("application/json", False), ("application/xml", True),))
+def test_content_type_conformance_integration(raw_schema, content_type, is_error):
+    schema = schemathesis.from_dict(raw_schema)
+    endpoint = schema.endpoints["/users"]["get"]
     case = models.Case(endpoint)
     response = make_response(content_type=content_type)
     if not is_error:

